_pages/en_US/finalizing-setup.txt

```diff
@@ -57,12 +61,6 @@ During this process, we also setup programs such as the following:
 
 #### Section II - Updating the System
 
-If, while following a previous version of this guide, you CTRTransfered your *New 3DS* to 2.1.0, you should now [restore your NAND backup](godmode9-usage#restoring-a-nand-backup) before doing this section.
-{: .notice--danger}
```

Comment:
Is this okay to remove?

Reply:
Resolved by re-word: "If your **New 3DS** was on firmware 2.1.0 before following this guide, you should [restore your NAND backup](godmode9-usage#restoring-a-nand-backup) before continuing."